packages/docs-app/src/examples/core-examples/breadcrumbsExample.tsx

```diff
@@ -48,13 +48,13 @@ const ITEMS: BreadcrumbProps[] = [
     { icon: "folder-close", text: "All files" },
     { icon: "folder-close", text: "Users" },
     { icon: "folder-close", text: "Janet" },
-    { href: "#", icon: "folder-close", text: "Photos" },
-    { href: "#", icon: "folder-close", text: "Wednesday" },
+    { href: "https://blueprintjs.com/docs/#core/components/overflow-list#", icon: "folder-close", text: "Photos" },
```

Comment:
this page may be served on different url paths (for example localhost or a /versions/4/ folder for different major versions), so let's drop everything before the `#`:

```suggestion
    { href: "#core/components/overflow-list", icon: "folder-close", text: "Photos" },
```